src/pages/ListPage/ListPage.tsx

```diff
@@ -210,7 +211,45 @@ const ListPage: React.FunctionComponent<unknown> = () => {
                         <title>Policies - Operations</title>
                     </Helmet>
                     <PageHeader>
-                        <PageHeaderTitle title={ Messages.pages.listPage.title } />
+                        <PageHeaderTitle title={ (
+                            <React.Fragment>
+                                {Messages.pages.listPage.title}
+                                <Popover
+                                    headerContent="About policies"
+                                    bodyContent={ (
+                                        <TextContent>
+                                            <Flex direction={ { default: 'column' } }>
+                                                <Text component={ TextVariants.p }>
+                                                    Policies evaluate system configurations in your environment,
+                                                    detect inventory changes and can send notifications when changes occur.
```

Comment:
This is a super small nitpick, but just for proper grammar- this adds a coma before the and 

`your environment, detect inventory changes, and can send `